docs/BUILDING_AN_INDEX.md

```diff
@@ -0,0 +1,61 @@
+# Building an Index Image
+
+Preflight's Operator policy (i.e. `preflight check operator ...`) requires
+access to an index image containing the Operator bundle under test.
+
+The fastest way to do this is to utilize the `opm` tool to build an index image,
+publish that image, and then provide that to Preflight when executing your
+checks. To install the `opm` cli, visit the
+[operator-framework/operator-registry
+releases](https://github.com/operator-framework/operator-registry/releases) page
+and download an appropriate binary for your system.
+
+## Building an Index With Your Bundle
+
+For reference, the below instructions were executed using `opm` at the following
+version:
+
+```shell
+$ opm version
+Version: version.Version{OpmVersion:"v1.18.0", GitCommit:"b826849", BuildDate:"2021-08-11T19:07:57Z", GoOs:"darwin", GoArch:"amd64"}
+```
+
+Your bundle must already be published to a registry in order to build your index
+image with that bundle.
+
+Run the following command to create an index with the specified bundle,
+substituting your bundle's registry path, and the desired registry path of your
+index. 
+
+(NOTE: Docker users may need to specify the `--container-tool=docker` flag to
+this command)
+
+```shell
+opm index add --bundles registry.example.org/your-namespace/your-bundle:0.0.1 --tag registry.example.org/your-namespace/your-index:0.0.1
+```
+
+(sample output)
+
+```shell
+INFO[0000] building the index                            bundles="[registry.example.org/your-namespace/your-bundle:0.0.1]"
+... truncated
+INFO[0002] [podman build -f index.Dockerfile359827617 -t registry.example.org/your-namespace/your-index:0.0.1 .]  bundles="[registry.example.org/your-namespace/your-bundle:0.0.1]"
+...
+
```

Comment:
shouldn't there be ````` to end the above shell?

Reply:
@acornett21 Whoops! Sorry about that - should be resolved now.